services/management/commands/turku_service_import/accessibility.py

```diff
@@ -0,0 +1,148 @@
+from munigeo.importer.sync import ModelSyncher
+
+from services.management.commands.turku_service_import.utils import get_ar_resource, get_ar_servicepoint_resource, \
+    get_ar_servicepoint_accessibility_resource
+from services.models import Unit, AccessibilityVariable, UnitAccessibilityProperty
+
+VERBOSITY = False
+LOGGER = None
+
+
+class AccessibilityImporter:
+    def __init__(self, verbosity=True, logger=None):
+        self.verbosity = verbosity
+        self.logger = logger
```

Comment:
We should probably get rid of that verbosity thing as it's not used anywhere. And maybe start using the logger? :)

Reply:
Yeah, lets do that